### PR TITLE
Integration of Resource utilization collection

### PR DESCRIPTION
**Pull request by akshaym99**
# Problem Statement
-  Problem statement

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [ ] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [ ] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

**Comment by akshaym99** [comment]
@priyankaborawake @kulrk please review

File: src/commons/utils/resource_util.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+"""Module to collect resource utilisation utils."""
+
+import logging
+from src.commons.utils import support_bundle_utils as remote_cmd
```
Comment (ketanarlulkar) [inline src/commons/utils/resource_util.py]: import execute_remote_command directly.

File: src/commons/utils/resource_util.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if node["node_type"] == "master":
+            host, user, passwd = node["hostname"], node["username"], node["password"]
+            resp = remote_cmd.execute_remote_command(cm_cmd.K8S_WORKER_NODES, host, user, passwd)
+            LOGGER.debug("response is: %s", str(resp))
+            resp = resp[1].read(-1).decode()
+            LOGGER.debug("response[1] is: %s", str(resp))
+            worker_node = resp.strip().split("\n")[1:]
+            LOGGER.info("worker nodes: %s", str(worker_node))
+    if action == "start":
+        remote_cmd.execute_remote_command(cm_cmd.CMD_YUM_NMON, "localhost", user, passwd)
```
Comment (ketanarlulkar) [inline src/commons/utils/resource_util.py]: Why do we want to execute local command remotely?

File: src/commons/utils/resource_util.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            resp = remote_cmd.execute_remote_command(cm_cmd.CMD_YUM_NMON, host, user, passwd)
+            LOGGER.debug("worker response: %s", str(resp))
+            resp = remote_cmd.execute_remote_command(cm_cmd.CMD_RUN_NMON, host, user, passwd)
+            LOGGER.debug("worker response: %s", str(resp))
+    else:
+        resp = remote_cmd.execute_remote_command(cm_cmd.CMD_KILL_NMON, "localhost", user, passwd)
+        LOGGER.debug("Local response: %s", str(resp))
+        resp = remote_cmd.execute_remote_command(cm_cmd.CMD_KILL_NMON, host, user, passwd)
+        LOGGER.debug("master response: %s", str(resp))
+        for worker in worker_node:
```
Comment (ketanarlulkar) [inline src/commons/utils/resource_util.py:90]: So are we just starting and stopping nmon tool on the nodes? Where is data collected? How can users view that? Why is that data not stored with execution logs file?

Comment (kulrk) [reply src/commons/utils/resource_util.py:90]: we can store in some remote location similar to collection of sb

File: src/commons/utils/resource_util.py (hunk 1 above)
Comment (akshaym99) [reply src/commons/utils/resource_util.py]: imported execute_remote_command directly.

File: src/commons/utils/resource_util.py (hunk 2 above)
Comment (akshaym99) [reply src/commons/utils/resource_util.py]: using run_local_cmd now with small change

File: src/commons/utils/resource_util.py (hunk 3 above)
Comment (akshaym99) [reply src/commons/utils/resource_util.py:90]: Integrating that logic here

Comment (akshaym99) [reply src/commons/utils/resource_util.py:90]: Added logic to move all nmon files to nfs_share location on client

**Comment by akshaym99** [comment]
log file added here:
https://jts.seagate.com/browse/CORTX-28261?focusedCommentId=2336623&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2336623

**Review by DhobaleVishal — APPROVED** [review]
LGTM, @akshaym99 fix codacy issues and attach execution logs.

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**